app/views/shelters/show.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<h1>Shelter Info</h1>
+
+<h3><a href="/shelters/<%= @shelter.id %>/edit">Update Shelter</a></h3>
+
+<form action=/shelters/<%= @shelter.id %> method="post">
```

Comment:
Explore creating a link either within form, or a link that will point to destroy route